geomstats/riemannian_metric.py

```diff
@@ -393,7 +393,7 @@ def optimal_quantization(self, points, n_centers=N_CENTERS,
 
         random_indices = gs.random.randint(low=0, high=n_points,
                                            size=(n_centers,))
-        centers = points[gs.ix_(random_indices, gs.arange(dimension))]
+        centers = points[gs.indexing(random_indices), :]
```

Comment:
Could we use the function's name gs.ix_ and match numpy's API?

Reply:
Replaced by cast.